src/ls/driver.ts

```diff
@@ -20,6 +21,9 @@ export default class YourDriverClass extends AbstractDriver<DriverLib, DriverOpt
 
   private queue = new PromiseQueue(1, Infinity);
 
+  // 100 queries, max age 1 minute
+  private cache = new LRUCache({ max: 100, maxAge: 1000*60 });
```

Comment:
The query result is cached for 1 minute, isn't too short? Will it result in many new questions being issued?

Reply:
Agreed, 1 minute is probably a bit short. It's a trade-off to make with the bad UX of not seeing a new table or column right when it is created, as there is no way for a user to force a cache refresh.

How about 10 minutes? Do you think that would be enough?

Reply:
Yeah, let's try 10 minutes.